README.md

```diff
@@ -11,28 +11,73 @@ This repo contains the frontend code for the dissolution web service. It's curre
 
 ## Recommendations
 
-We recommend the use of Visual Studio Code for development as it allows the installation of the TSLint and the Nunjucks plugins. These plugins will make linting of TS and Nunjuck much better than mmost code editors.
+We recommend the use of [Visual Studio Code](https://code.visualstudio.com/) for development as it allows the installation of the [TSLint](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin) and the [Nunjucks](https://marketplace.visualstudio.com/items?itemName=ronnidc.nunjucks) plugins. These plugins will make linting of TS and Nunjuck much better than mmost code editors.
 
 IntelliJ does not have a Nunjuck plugin but you can configure IntelliJ to provide syntax highlighting using Twig plugin
 
 ## Running locally with Docker  
 
-Required tools:
-- [Docker for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
-- [Docker-Compose](https://docs.docker.com/compose/install/)
+1. Required tools:
+    - [Docker for Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+    - [Docker-Compose](https://docs.docker.com/compose/install/)
 
-Once installed, open Docker for Mac and navigate to Preferences -> Resources and change the settings to Memory => 4 GB, CPUs 4, SWAP = 1 GB.
+2. Once installed, open Docker for Mac and navigate to Preferences -> Resources and change the settings to Memory => 4 GB, CPUs 4, SWAP = 1 GB.
 
-Add `127.0.0.1 chs-dev cdn.chs-dev account.chs-dev` to your `/etc/hosts` file.
+3. Add the following to your `/etc/hosts` file:
 
-Login to the Companies House AWS account and run the following command in a fresh terminal session:
+    ```
+    127.0.0.1 chs-dev cdn.chs-dev account.chs-dev chs-company-profile-api
+    ```
 
-`docker login -u AWS -p "$(aws ecr get-login-password)" https://169942020521.dkr.ecr.eu-west-1.amazonaws.com`
+4. Login to the Companies House AWS account and run the following command in a fresh terminal session:
 
-To bring the environment up, run `docker-compose up` in the project folder.
+    ```
+    $ docker login -u AWS -p "$(aws ecr get-login-password)" https://169942020521.dkr.ecr.eu-west-1.amazonaws.com
+    ```
 
-After making local changes to the app, Ctrl+C on the running `docker-compose` terminal session and re-run `docker-compose up` command.
+5. Link your GitHub SSH key to to `dissolution-web` directory to access private dependencies in Docker:
 
-Make local changes to the app, Ctrl/+C on the running `docker-compose` terminal session and re-run the command.
+    ```
+    $ mkdir .ssh
+    $ ln ~/.ssh/id_rsa .ssh/id_rsa
+    ```
+    
+6. Clone [Dissolution API](https://github.com/companieshouse/dissolution-api) and [Company Profile API](https://github.com/companieshouse/chs-company-profile-api) and ensure you have the following folder structure locally:
```

Comment:
@naufalmirk because we aren't going to be pushing containers of the dissolution-api (for now), I've suggested we setup our dev env folders in a way that we can point directly to the Dissolution API dockerfile.

Also, the Dockerfile created by the Tiny Rebels for the Company Profile API needs AWS creds to grab the relevant files from S3 to build the Perl app. Therefore, I've made this sometime that the devs will have to setup themselves for now as, again, we don't have a dedicated AWS user for Docker.

Thoughts? happy to proceed for now with a view to tidy this up in future considering this is for local dev only?

Reply:
we don't need to build the container, since the container will be pulled from the ECR repo, and has been built locally.

Reply:
I've been using the container that the Tiny Rebels prepared for this integration. I'm not sure if it's in ECR. If it is, then great, we can remove the need to manually clone the chs-company-profile-api repo. But I do think we'll need the Dissolution API to be cloned to a specific location as we aren't regularly pushing containers to ECR for it yet.